flocker/node/test/test_script.py

```diff
@@ -90,6 +90,8 @@ def get_all_ips():
     """
     Find all IPs for this machine.
 
+    XXX: Move this to flocker.common.???
+
```

Comment:
Hm.  This is really `get_all_ipv4s`.  Maybe put a real `get_all_ips` (ipv4 and ipv6) in `flocker.common` and we can think about transitioning everything to that and away from this.

You're planning to include ipv6 addresses in the matching, right?